packages/lib-classifier/src/plugins/tasks/survey/components/SurveyTask.spec.js

```diff
@@ -264,8 +260,8 @@ describe('SurveyTask', function () {
 
         it('should remove filters on Clear Filters button click (within Chooser)', async function () {
           const user = userEvent.setup({ delay: null })
-          const { getAllByRole, getByRole, getByTestId } = render(<DefaultStory />)
-          const filterButton = getByRole('button', { name: 'SurveyTask.CharacteristicsFilter.filter' })
+          const { getAllByRole, getByTestId, getByText } = render(<DefaultStory />)
+          const filterButton = getByText('SurveyTask.CharacteristicsFilter.filter')
           await user.click(filterButton)
           const cowHorseFilterButton = getByTestId('filter-LK-CWHRS')
```

Comment:
This could use `screen.getByLabelText` too, since each icon should also have a text label.